lib/network.sh

```diff
@@ -7,6 +7,16 @@
 #    - Which variable to write the result into
 #    - Network CIDR (e.g. 172.22.0.0/24)
 #    - Which address (e.g. first, second, etc)
+
+# Set update-alternatives to python3
+if [[ ${OS_VERSION_ID} == "18.04" ]]; then
+  sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.6 1
+elif [[ ${OS_VERSION_ID} == "20.04" ]];
+  sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1
+else
+  sudo alternatives --set python /usr/bin/python3
+fi
+
```

Comment:
These should not be here IMO. This is not related to networking and any installation should be a part of 01 or 02 not lib/ IMO

Reply:
Fixed.